onchain/src/social/account.cairo

```diff
@@ -95,6 +120,9 @@ mod tests {
         ISocialAccountSafeDispatcherTrait
     };
 
+    const public_key: u256 = 45;
+    const ISRC5_ID: felt252 = 0x2717e95370c20103b33f13a20ed0f89783312fd08860f522ef8f6177fedc5e0;
```

Comment:
Why not use `ISRC5_ID` that is already declared above?

Reply:
Yes, I think we should keep the interface structure for generating the ID in a separate file. 
So we would update it when there's any change to the interface.

Also about the ID being in different places, I couldn't access it in the test, that's why I also declared it in the test block.

Reply:
@maciejka, where would you want me to put the interface structure cairo file?

Reply:
> @maciejka, where would you want me to put the interface structure cairo file?

yes, please

Reply:
> Also about the ID being in different places, I couldn't access it in the test, that's why I also declared it in the test block.

Why not `use super::ISRC5_ID;`?

Reply:
I mean in which file or folder should I put it in?

Reply:
> > Also about the ID being in different places, I couldn't access it in the test, that's why I also declared it in the test block.
> 
> Why not `use super::ISRC5_ID;`?

Oh yes, I will try that.

Reply:
@maciejka, I have made updates.
Please check again 🙂